src/packages/block/block/modals/block-catalogue/block-catalogue-modal.element.ts

```diff
@@ -57,14 +82,9 @@ export class UmbBlockCatalogueModalElement extends UmbModalBaseElement<
 		this._openClipboard = this.data.openClipboard ?? false;
 		this._blocks = this.data.blocks ?? [];
 		this._blockGroups = this.data.blockGroups ?? [];
-	}
 
-	/*
-	#onClickBlock(contentElementTypeKey: string) {
-		this.modalContext?.updateValue({ key: contentElementTypeKey });
-		this.modalContext?.submit();
+		this.#itemManager.setUniques(this._blocks.map((x) => x.contentElementTypeKey));
 	}
-	*/
 
```

Comment:
It would be nice to render the headlines to they look like Current BO.

Also notice in current backoffice, if you only see one group, we do not render the group headline. :-)